lte/gateway/c/oai/lib/directoryd/GatewayDirectorydClient.cpp

```diff
@@ -57,22 +57,26 @@ GatewayDirectoryServiceClient::GatewayDirectoryServiceClient() {
 bool GatewayDirectoryServiceClient::UpdateRecord(
     const std::string& id, const std::string& location,
     std::function<void(Status, Void)> callback) {
-  GatewayDirectoryServiceClient& client = get_instance();
-
   UpdateRecordRequest request;
-  Void response;
-
   request.set_id(id);
   request.set_location(location);
+  return GatewayDirectoryServiceClient::updateRecordImpl(request, callback);
```

Comment:
nit: remove the `GatewayDirectoryServiceClient` context here? since the function it's on the same namespace